Error: The `form` does not need to be labelled by the `legend` as the `fieldset` & `legend` combination already provides those semantics.
Error: The `form` element was not requested.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: The prompt did not request a radio be checked by default.
diff --git a/output/Radio Group/4/efce7b1f-f9c8-442d-a2ce-e1e9c00b13f7.html b/output/Radio Group/4/efce7b1f-f9c8-442d-a2ce-e1e9c00b13f7.html
index 28212bb..8926ef2 100644
--- a/output/Radio Group/4/efce7b1f-f9c8-442d-a2ce-e1e9c00b13f7.html	
+++ b/output/Radio Group/4/efce7b1f-f9c8-442d-a2ce-e1e9c00b13f7.html	
@@ -1,14 +1,12 @@
-<form aria-labelledby="theme-chooser-label">
-  <fieldset>
-    <legend id="theme-chooser-label">Choose a theme</legend>
-    <label>
-      <input type="radio" name="theme" value="light" checked> Light
-    </label>
-    <label>
-      <input type="radio" name="theme" value="dark"> Dark
-    </label>
-    <label>
-      <input type="radio" name="theme" value="high-contrast"> High Contrast
-    </label>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>Choose a theme</legend>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light"> Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark"> Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast"> High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
